upstream-community-operators/tektoncd-pipeline-operator/tektoncd-pipeline-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,248 @@
+          - installs
+          verbs:
+          - '*'
+        - apiGroups:
+          - rbac.authorization.k8s.io
+          resources:
+          - clusterroles
+          verbs:
+          - get
+          - create
+          - update
+        - apiGroups:
+          - ""
+          resources:
+          - serviceaccounts
+          verbs:
+          - get
+          - list
+          - create
+          - update
+          - delete
+          - patch
+          - watch
+        - apiGroups:
+          - rbac.authorization.k8s.io
+          resources:
+          - clusterrolebindings
+          verbs:
+          - get
+          - create
+          - update
+        - apiGroups:
+          - apiextensions.k8s.io
+          resources:
+          - customresourcedefinitions
+          verbs:
+          - get
+          - create
+          - update
+        - apiGroups:
+          - admissionregistration.k8s.io
+          resources:
+          - mutatingwebhookconfigurations
+          verbs:
+          - get
+          - list
+          - create
+          - update
+          - delete
+          - patch
+          - watch
+        - apiGroups:
+          - build.knative.dev
+          resources:
+          - builds
+          - buildtemplates
+          - clusterbuildtemplates
+          verbs:
+          - get
+          - list
+          - create
+          - update
+          - delete
+          - patch
+          - watch
+        - apiGroups:
+          - extensions
+          resources:
+          - deployments
+          verbs:
+          - get
+          - list
+          - create
+          - update
+          - delete
+          - patch
+          - watch
+        - apiGroups:
+          - extensions
+          resources:
+          - deployments/finalizers
+          verbs:
+          - get
+          - list
+          - create
+          - update
+          - delete
+          - patch
+          - watch
+        serviceAccountName: tektoncd-pipeline-operator
+      deployments:
+      - name: tektoncd-pipeline-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: tektoncd-pipeline-operator
+          strategy: {}
+          template:
+            metadata:
+              labels:
+                name: tektoncd-pipeline-operator
+            spec:
+              containers:
+              - command:
+                - tektoncd-pipeline-operator
+                env:
+                - name: WATCH_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.annotations['olm.targetNamespaces']
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: OPERATOR_NAME
+                  value: tektoncd-pipeline-operator
+                image: quay.io/openshift-pipeline/nikthoma_tektoncd-pipeline-operator:v0.2.0-1
```

Comment:
Update image ref to `openshift-pipeline/tektoncd-pipeline-operator`

Reply:
updated:
quay.io/openshift-pipeline/openshift-pipelines-operator